docs/pages/versions/unversioned/index.mdx

```diff
@@ -24,22 +24,21 @@ import { Gyroscope } from 'expo-sensors';
 
 This allows you to write [`Contacts.getContactsAsync()`](/versions/latest/sdk/contacts#contactsgetcontactsasynccontactquery) and read the contacts from the device, read the gyroscope sensor to detect device movement, or start the phone's camera and take photos.
 
-## These packages work in bare React Native apps too
+## All Expo SDK packages work in any React Native app
 
-The easiest way to create a bare React Native app with support for the Expo SDK is by running the command:
+...and of course they do, because Expo apps are React Native apps! The easiest way to create a React Native app with support for Expo SDK packages is to use `create-expo-app`, but you can also add Expo SDK support to an existing React Native app with the `npx install-expo-modules` command.
```

Comment:
```suggestion
Expo apps are React Native apps! The easiest way to create a React Native app with support for Expo SDK packages is to use `create-expo-app`. However, you can also add Expo SDK support to an existing React Native app with the `npx install-expo-modules` command.
```

Please, let's not use ellipses.

Reply:
can you elaborate? I don't see any mention of this in the docs style guide so I'm curious about the reasoning